### ISSUE TITLE
Changing the webroot via the config file

### ISSUE THREAD

**Issue by UltraBlackLinux** [roundcube/roundcubemail#9152]
Hey there,
In my opinion services on the same server should be hosted the same (sub)domain, which is also how I have set up a few other services, but roundcube and another service have me stumped. They have no way to change the webroot, meaning I cannot use example.com/roundcube to access roundcube, because it doesn't use that webroot.

How would I go dealing with this? I have seen some weird solutions with moving the files from /var/www/html to /var/www/html/roundcube to be able to access roundcube on /roundcube, however I don't think that's such easy task on the docker image, as it's much more difficult to change configurations on the fly. The example also used apache, but the docker image uses nginx, so I'm unsure about what to do.

When I saw the example php-fpm docker compose file I thought that I might be able to change the webroot by running php-fpm on the changed webroot somehow, but that sadly didn't help either.

Did I miss something? If not, can I expect a webroot configuration option to land in roundcube at all?

**Comment by alecpl** [comment]
I'm not sure what your issue is, but maybe you need to set `$config['request_path']`. Take a look into defaults.inc.php for more info.

If your problem is with the docker image. Goto https://github.com/roundcube/roundcubemail-docker/issues

**Comment by UltraBlackLinux** [comment]
Oh thank you so much for pointing me there! I didn't know the defaults explained everything, I'd expected a config file with almost nothing in it.

In the meantime, I have figured out a way to make roundcube appear on the path I want it to appear on. Apparently one missing slash caused caddy to not remove the prefix from the request.

There is however an issue: Roundcube apparently completely ignores the request path (or it does not what I'd expect it to do), and still redirects to / instead of /webroot/ after logging in (and the login fails? I got in once thanks to persistent login, but my mailbox was empty and I got some errors)
